tests/es6modules/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1378,6 +1400,14 @@ export default (function() {
 "
 `;
 
+exports[`test export_default_function_expression_named.js 1`] = `
+"export default (function f(){})
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+export default (function f() {
+})
```

Comment:
I think you want to rebase, then delete [this line](https://github.com/jlongster/prettier/blob/237e6f44bc75269b96b31541c4a06ad293a591dd/src/printer.js#L1832) so you get a semicolon here.